polyfill/test/duration.mjs

```diff
@@ -212,6 +212,16 @@ describe('Duration', () => {
       equal(`${new Duration(0, 0, 0, 0, 0, 0, 0, 1111, 1111, 1111)}`, 'PT1.112112111S');
       equal(`${Duration.from({ seconds: 120, milliseconds: 3500 })}`, 'PT123.500S');
     });
+    it('negative sub-second units are balanced correctly', () => {
+      equal(`${Duration.from({ milliseconds: -250 })}`, '-PT0.250S');
+      equal(`${Duration.from({ milliseconds: -3500 })}`, '-PT3.500S');
+      equal(`${Duration.from({ milliseconds: -250 })}`, '-PT0.250S');
+      equal(`${Duration.from({ milliseconds: -3500 })}`, '-PT3.500S');
+      equal(`${Duration.from({ milliseconds: -250 })}`, '-PT0.250S');
+      equal(`${Duration.from({ milliseconds: -3500 })}`, '-PT3.500S');
```

Comment:
why are the same two test cases repeated thrice?

Reply:
Oops! That was intended to be µs and ns as well.